lua/orgmode/api/init.lua

```diff
@@ -1,13 +1,19 @@
 ---@diagnostic disable: invisible
 local OrgFile = require('orgmode.api.file')
 local OrgHeadline = require('orgmode.api.headline')
+local Hyperlinks = require('orgmode.org.hyperlinks')
+local Link = require('orgmode.org.hyperlinks.link')
 local orgmode = require('orgmode')
 
 ---@class OrgApiRefileOpts
 ---@field source OrgApiHeadline
 ---@field destination OrgApiFile | OrgApiHeadline
 
 ---@class OrgApi
+---@field load fun(name?: string|string[]): OrgApiFile|OrgApiFile[]
```

Comment:
These types are not necessary. Methods should appear in the types.

Reply:
The reason was to make it easy to annotate the API module in a different plugin to help development. See [here](https://github.com/lyz-code/telescope-orgmode.nvim/pull/16/commits/c1492a655685f097886c0dc3edc3603dedd7d2cd#r1608568226)
Is there a way to tell the LSP where to find them without copying?

Reply:
I removed the type hints here, but I will keep them for now in Telescope-orgmode.